src/Requests/CollectionTtl.php

```diff
@@ -0,0 +1,46 @@
+<?php
+declare(strict_types=1);
+
+namespace Momento\Requests;
+
+class CollectionTtl
+{
+    private ?int $ttl;
```

Comment:
If we are using integers to represent ttls, the variable names need to include the unit for clarity.  I think we were using `ttlSeconds` in the previous code.